seed/data_importer/tests/integration/test_footprints_import.py

```diff
@@ -152,21 +152,16 @@ def test_demo_v2(self):
 
         self.assertEqual(len(ps), 3)
 
```

Comment:
Why'd these changes happen here?

Reply:
When property_1. has a null conditioned. floor area, because the rule was set to check for range, not a non-null value, no error detailed message is generated for property 1. This test is no longer valid. Part of the reason why I edited the test prior to removing it.

Reply:
Reverted test as I misunderstood the original test.